src/components/Categories.jsx

```diff
@@ -1,9 +1,23 @@
+import { useDispatch } from 'react-redux';
 import styles from '../sass/Categories.module.scss';
+import { checkStatus } from '../redux/categories/categoriesSlice';
 
-const Categories = () => (
-  <div className={styles.btn_wrapper}>
-    <button type="button" className={styles.check_btn}>Check Status</button>
-  </div>
-);
+const Categories = () => {
+  const dispatch = useDispatch();
+
+  return (
+    <div className={styles.btn_wrapper}>
+      <button
+        type="button"
+        className={styles.check_btn}
+        onClick={() => {
+          dispatch(checkStatus());
```

Comment:
* Kindly look into this bug here as when I click the check status button nothing happens, Kindly ensure that the `under construction` text shows up when the button is clicked as you have written the slice already 👍

Reply:
Thank you, @Davidosky007

Reply:
thank you ! @HAMMAS-SALEEM